common/source/docs/common-transmitter-tuning.rst

```diff
@@ -271,3 +274,145 @@ At that point you can land the vehicle, or just enjoy flying it.
++--------+-------------------------+----------------------------------------------------------------------+
+|46      |Rate Pitch kP            | :ref:`ATC_RAT_PIT_P<ATC_RAT_PIT_P__AC_AttitudeControl_Multi>`        |
++--------+-------------------------+----------------------------------------------------------------------+
+|47      |Rate Pitch kI            | :ref:`ATC_RAT_PIT_I<ATC_RAT_PIT_I__AC_AttitudeControl_Multi>`        |
++--------+-------------------------+----------------------------------------------------------------------+
+|48      |Rate Pitch kD            | :ref:`ATC_RAT_PIT_D<ATC_RAT_PIT_D__AC_AttitudeControl_Multi>`        |
++--------+-------------------------+----------------------------------------------------------------------+
+|49      |Rate Roll kP             | :ref:`ATC_RAT_RLL_P<ATC_RAT_RLL_P__AC_AttitudeControl_Multi>`        |
++--------+-------------------------+----------------------------------------------------------------------+
+|50      |Rate Roll kI             | :ref:`ATC_RAT_RLL_I<ATC_RAT_RLL_I__AC_AttitudeControl_Multi>`        |
++--------+-------------------------+----------------------------------------------------------------------+
+|51      |Rate Roll kD             | :ref:`ATC_RAT_RLL_D<ATC_RAT_RLL_D__AC_AttitudeControl_Multi>`        |
++--------+-------------------------+----------------------------------------------------------------------+
+|52      |Rate Pitch FF            | :ref:`ATC_RAT_PIT_FF<ATC_RAT_PIT_FF>` (heli only)                    |
++--------+-------------------------+----------------------------------------------------------------------+
+|53      |Rate Roll FF             | :ref:`ATC_RAT_RLL_FF<ATC_RAT_RLL_FF>` (heli only)                    |
++--------+-------------------------+----------------------------------------------------------------------+
+|54      |Rate Yaw FF              | :ref:`ATC_RAT_YAW_FF<ATC_RAT_YAW_FF>` (heli only)                    |
++--------+-------------------------+----------------------------------------------------------------------+
+|57      |Winch                    | :ref:`WINCH__RATE_MAX<WINCH__RATE_MAX>`                              |
++--------+-------------------------+----------------------------------------------------------------------+
+|58      |SysID Magnitude          | :ref:`SIDS_MAGNITUDE<SID_MAGNITUDE>`                                 |
++--------+-------------------------+----------------------------------------------------------------------+
+
+
+These values can be either set manually or using Mission Planner
+
+
+Setting with Mission Planner
+============================
+
+Rate Roll P and Rate Pitch P will be used in the following example procedure
+
+.. image:: ../images/RollPitchTuning.png
+    :target: ../_images/RollPitchTuning.png
+
+#. Connect your autopilot to Mission Planner
+#. On Mission Planner, select Config/Tuning >> Copter Pids
```

Comment:
I think the MP page is Config >> Extended Tuning  now..

Reply:
it is...will update and then merge...thanks